server/src/services/listings.ts

```diff
@@ -35,4 +42,17 @@ const getAllListingsWithIds = async (
 const getListing = async (listingId: number): Promise<ListingResponse> =>
   listingStorage.getListing(listingId);
 
+const updateListingSuccess = async (listingId: number) => {
```

Comment:
call it updateListingStatus?

Reply:
Or maybe "updateListingOutcomeStatus" to show that "successful/unsuccessful" status are some kind of terminal state of a listing status.

Reply:
@ifhuang @xlai20 how about `updateListingSuccessStatus`?

Reply:
I think the word "Success" is better to be avoided because your new status can be "unsuccessful" as well. All these are fine:
 - updateListingStatus
 - updateListingTerminalStatus
 - updateListingOutcome
 - updateListingOutcomeStatus

Reply:
@xlai20 I thought 'success' means both 'successful' and 'unsuccessful'. Similar to like if you ask 'What is your level of satisfaction?', then the answer can be 'Satisfied' or 'Unsatisfied'. But if it's confusing, I can change to 'updateListingOutcomeStatus'.

Reply:
+1 to avoid the word 'success', it's either unnecessary or confusing to have 'SuccessStatus' compared to just 'Status'.

Reply:
@chshda Okay! I'll change to 'updateListingOutcomeStatus' then.